example/shared/css/index.css

```diff
@@ -171,4 +171,11 @@ ul li:nth-last-child(1) {
     #your-products {
         margin-top: 1em;
     }
+    footer {
+        margin-top: 1em;
+        padding-top: 1em;
+        border-width: 1px 0 0 0;
```

Comment:
Suggest collapsing the width, style and default colour and then override the bits that are different in separate rules. 

Are the additional colours needed if they're 0px wide?